packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts

```diff
@@ -48,17 +51,25 @@ export class HorizontalBarChart extends FASTElement {
 
   public rootDiv!: HTMLDivElement;
   public chartContainer!: HTMLDivElement;
+  public elementInternals: ElementInternals = this.attachInternals();
 
   private _isRTL: boolean = false;
   private barHeight: number = 12;
   private _bars: SVGRectElement[] = [];
 
+  constructor() {
+    super();
+
+    this.elementInternals.role = 'region';
+  }
+
   connectedCallback() {
     super.connectedCallback();
 
     validateChartPropsArray(this.data, 'data');
 
     this._isRTL = getRTL(this);
+    this.elementInternals.ariaLabel = this.chartTitle || `Horizontal bar chart with ${this.data.length} bars.`;
```

Comment:
the data is a 2d object, first representing each row and second shows bars in each row. Which property are you using

Reply:
the first one. it basically announces the total number of bars (or rows) present.

Reply:
Then call it - Horizontal bar chart with ${this.data.length} categories.

Reply:
done